Bio/SeqUtils/ProtParam.py

```diff
@@ -189,9 +189,23 @@ def flexibility(self):
 
         return scores
 
-    def gravy(self):
-        """Calculate the gravy according to Kyte and Doolittle."""
-        total_gravy = sum(ProtParamData.kd[aa] for aa in self.sequence)
+    def gravy(self, scale="KyteDoolitle"):
+        """Calculate the GRAVY (Grand Average of Hydropathy) according to Kyte and Doolitle, 1982.
+
+        Utilizes the given Hidrophobicity scale, by default uses the original proposed
+        by Kyte and Doolittle (KyteDoolitle). Other options are: Aboderin, AbrahamLeo,
+        Argos, BlackMould, BullBreese, Casari, Cid, Cowan3.4, Cowan7.5, Eisenberg,
+        Engelman, Fasman, Fauchere, GoldSack, Guy, Jones, Juretic, Kidera, Miyazawa,
+        Parker,Ponnuswamy, Rose, Roseman, Sweet, Tanford, Wilson and Zimmerman.
+
+        New scales can be added in ProtParamData.
+        """
+        selected_scale = ProtParamData.gravy_scales.get(scale, -1)
+
+        if selected_scale == -1:
+            raise ValueError("scale: {} not know".format(scale))
```

Comment:
Typo: know -> known

Style: Please use an f-string rather than the format method

Ideally there would be a simple test for the unknown scale, using ``.assertRaises``